sdks/java/io/amazon-web-services2/build.gradle

```diff
@@ -75,6 +75,5 @@ test {
       '--awsRegion=us-west-2',
       '--awsCredentialsProvider={"@type": "StaticCredentialsProvider", "accessKeyId": "key_id_value", "secretAccessKey": "secret_value"}'
   ])
-  // Forking every test resolves an issue where KinesisMockReadTest gets stuck forever.
-  forkEvery 1
+  maxParallelForks 4
```

Comment:
Can we make sure that the previous issue with KinesisMockReadTest is not a case anymore?

Reply:
I couldn't find any issue testing locally, `KinesisMockReadTest` was just running fine.
Unfortunately there's also no ticket linked in the commit making it hard to know what previously happened.

Reply:
It's just got stuck forever for unknown reasons. Though, if there is no issue for now then probably it was "automatically" fixed with a new version of SDK or Beam.

Reply:
Let's wait for `Java PreCommit` finish.